moment-timezone.js

```diff
@@ -318,6 +318,15 @@
 	}
 
 	function rebuildGuess () {
+
+		// use Intl API when available and returning valid time zone
+		if (typeof Intl === 'object' && isFunction(Intl.DateTimeFormat)) {
+			var zone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+			if (zone && (zone.indexOf("/") > -1 || zone === 'UTC')) {
```

Comment:
How about something like `if (typeof zone === 'string' && names[normalizeName(zone)]) {` to make sure a identifier returned from `Intl` actually has data loaded? Maybe we could add a `logError` if `Intl` returns an identifier that was not loaded in the data set?